pkg/minikube/assets/addons.go

```diff
@@ -782,22 +778,21 @@ func SelectAndPersistImages(addon *Addon, cc *config.ClusterConfig) (images, cus
 		cc.CustomAddonRegistries = mergeMaps(cc.CustomAddonRegistries, customRegistries)
 	}
 
-	err = nil
 	// If images or registries were specified, save the config afterward.
 	if viper.IsSet(config.AddonImages) || viper.IsSet(config.AddonRegistries) {
 		// Since these values are only set when a user enables an addon, it is safe to refer to the profile name.
-		err = config.Write(viper.GetString(config.ProfileName), cc)
 		// Whether err is nil or not we still return here.
+		return images, customRegistries, config.Write(viper.GetString(config.ProfileName), cc)
 	}
-	return images, customRegistries, err
+	return images, customRegistries, nil
```

Comment:
Since we removed the `err` variable we can also remove the variable name from the return signature on L735.

Reply:
Yes, set variable name for error to _, rather than remove altogether since we want to keep images, customRegistries.